client/packages/system/src/Report/components/ReportSelector.tsx

```diff
@@ -82,10 +83,10 @@ export const ReportSelector: FC<PropsWithChildren<ReportSelectorProps>> = ({
   const options: SplitButtonOption<string>[] = useMemo(() => {
     const reports = data
       ? data?.nodes?.map(report => ({
-          value: report.id,
-          label: report.name,
-          isDisabled: disabled,
-        }))
+        value: report.id,
+        label: report.isCustom ? report.name : t(getReportKey(report.name)),
```

Comment:
I liked having both functions here as for me it is more clear that the standard translation function is still being used without reading through the hook